### ISSUE TITLE
Questionable Protocol Choice for ElevenLabs.

### ISSUE THREAD

**Issue by yeki6110** [ccappetta/bidirectional_streaming_ai_voice#4]
Hi! This is just more of a question.

ElevenLabs provides Websockets for TTS.
https://elevenlabs.io/docs/api-reference/websockets

What reasons exist for favoring the use of an asynchronous HTTP client over Websockets in this circumstance?

**Comment by ccappetta** [comment]
This is a good question I've actually just put some more comments about the elevenlabs websockets in the readme. But summarizing - I started out using elevenlabs websockets and was trying it with MVP and VLC libraries before I settled on pygame. I was getting mixed results. I do recall that I needed the anthropic streaming to be synchronous so thats why the audio generation and playback became asynchronous. I don't remember if that's WHY I left the websockets functionality behind for a 'regular old' api call but it might have been.  

It could well be that the websockets feature could work better with the async design with pygame than it did with MVP and VLC
